crates/rpc/rpc-builder/tests/it/utils.rs

```diff
@@ -26,7 +27,8 @@ pub fn test_address() -> SocketAddr {
 pub async fn launch_auth(secret: JwtSecret) -> AuthServerHandle {
     let config = AuthServerConfig::builder(secret).socket_addr(test_address()).build();
     let (tx, _rx) = unbounded_channel();
-    let beacon_engine_handle = BeaconConsensusEngineHandle::<EthEngineTypes>::new(tx);
+    let listeners: EventListeners<BeaconConsensusEngineEvent> = Default::default();
+    let beacon_engine_handle = BeaconConsensusEngineHandle::<EthEngineTypes>::new(tx, listeners);
```

Comment:
no need to assign listeners here right? 
```suggestion
    let beacon_engine_handle = BeaconConsensusEngineHandle::<EthEngineTypes>::new(tx, Default::default());
```